include/wlr/xwayland.h

```diff
@@ -119,6 +119,13 @@ struct wlr_xwayland_surface_size_hints {
 	uint32_t win_gravity;
 };
 
+enum wlr_xwayland_icccm_input_state {
```

Comment:
Can we add a link to https://www.x.org/releases/X11R7.6/doc/xorg-docs/specs/ICCCM/icccm.html#input_focus here so that people can figure out what each model means?